apps/mymedicare_cb/authorization.py

```diff
@@ -280,14 +280,25 @@ def service_health_check(self, request):
         the BB2 /health/external check.
         """
         headers = self.slsx_common_headers(request)
-
-        response = requests.get(
-            self.healthcheck_endpoint,
-            headers=headers,
-            allow_redirects=False,
-            verify=self.verify_ssl_internal,
-            timeout=5,
-        )
+        max_retries = 3
+        retries = 0
+        while retries < max_retries:
+            try:
+                response = requests.get(
+                    self.healthcheck_endpoint,
+                    headers=headers,
+                    allow_redirects=False,
+                    verify=self.verify_ssl_internal,
+                    timeout=10,
+                )
+                response.raise_for_status()
+                return True
+            except requests.exceptions.RequestException:
+                if retries < max_retries:
+                    print(f"Request failed. Retrying... ({retries+1}/{max_retries})")
+                    retries += 1
+                else:
+                    return False
         response.raise_for_status()
         return True
```

Comment:
This looks like unreachable code with the returns in the try/catch, I think we can safely remove these two lines.
```suggestion
```